packages/grpc-js/test/test-server.ts

```diff
@@ -116,7 +116,7 @@ describe('Server', () => {
 
       assert.throws(() => {
         server.bindAsync('localhost:0', null as any, noop);
-      }, /creds must be an object/);
+      }, /creds must be a ServerCredentials object/);
```

Comment:
Maybe we should have another test for an object of the wrong type being passed?

Reply:
I added another test that matches the problem the person who filed the linked issue had.